UploadFile/Table/Tab50410.BCPUpload.al

```diff
@@ -0,0 +1,56 @@
+table 50410 "BCP Upload"
```

Comment:
I think upload is not best name for table which contain files. Maybe something like "BCP Content"?

Reply:
i'm changing it